.github/workflows/scala-steward.yml

```diff
@@ -0,0 +1,18 @@
+# This workflow will launch at 00:00 every Sunday
```

Comment:
Needs an apache license to pass the rat check.

Reply:
Yeah missed that initially, I added it